### ISSUE TITLE
Uncaught TypeError: Cannot read property 'append' of undefined

### ISSUE THREAD

**Issue by proto-type** [atom/atom-dark-syntax#9]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.169.0
**System**: Unknown Windows Version
**Thrown From**: [atom-dark-syntax](https://github.com/atom/atom-dark-syntax) package, v0.23.0

### Stack Trace

Uncaught TypeError: Cannot read property 'append' of undefined

```
At c:\Users\Blue Lin\AppData\Local\atom\app-0.169.0\resources\app\node_modules\settings-view\lib\themes-panel.js:204

TypeError: Cannot read property 'append' of undefined
  at c:\Users\Blue Lin\AppData\Local\atom\app-0.169.0\resources\app\node_modules\settings-view\lib\themes-panel.js:204:36
  at c:\Users\Blue Lin\AppData\Local\atom\app-0.169.0\resources\app\node_modules\emissary\lib\emitter.js:155:30
  at Array.forEach (native)
  at PackageManager.module.exports.Emitter.emitSlow (c:\Users\Blue Lin\AppData\Local\atom\app-0.169.0\resources\app\node_modules\emissary\lib\emitter.js:154:47)
  at PackageManager.module.exports.Emitter.emit (c:\Users\Blue Lin\AppData\Local\atom\app-0.169.0\resources\app\node_modules\emissary\lib\emitter.js:107:30)
  at PackageManager.module.exports.PackageManager.emitPackageEvent (c:\Users\Blue Lin\AppData\Local\atom\app-0.169.0\resources\app\node_modules\settings-view\lib\package-manager.js:368:19)
  at c:\Users\Blue Lin\AppData\Local\atom\app-0.169.0\resources\app\node_modules\settings-view\lib\package-manager.js:284:19
  at exit (c:\Users\Blue Lin\AppData\Local\atom\app-0.169.0\resources\app\node_modules\settings-view\lib\package-manager.js:43:16)
  at BufferedProcess.triggerExitCallback (c:\Users\Blue Lin\AppData\Local\atom\app-0.169.0\resources\app\src\buffered-process.js:63:47)
  at ChildProcess.<anonymous> (c:\Users\Blue Lin\AppData\Local\atom\app-0.169.0\resources\app\src\buffered-process.js:85:18)
  at ChildProcess.emit (events.js:110:17)
  at Process.ChildProcess._handle.onexit (child_process.js:1068:12)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

/cc @atom/core

**Comment by kevinsawicki** [comment]
This is a duplicate of atom/settings-view#322 and was fixed in the 0.170 Atom release

**Comment by benogle** [comment]
Still not sure why it gets the package wrong in some cases...
